.travis.yml

```diff
@@ -52,4 +52,4 @@ deploy:
   - provider: script
     on:  # yamllint disable-line rule:truthy
       all_branches: true
-    script: ./deploy.sh
+    script: ./deploy.sh || travis_terminate 1;
```

Comment:
I think we don't need here,  as this will be executed after merging PR
@rootfs can you confirm

Reply:
thats correct. I removed it and updated the patchset.